sql/updates/latest-dev.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE FUNCTION ts_realtime_agg_update() returns void LANGUAGE PLPGSQL AS
```

Comment:
I suggest an anonymous code block instead of creating, executing, and then immediately dropping the function: https://www.postgresql.org/docs/current/sql-do.html

Reply:
Any reason to not do what I suggested? It would be cleaner...